WordPress/Classes/ViewRelated/Views/List/NotificationsList/NotificationsTableViewCellContent.swift

```diff
@@ -0,0 +1,122 @@
+import SwiftUI
+import DesignSystem
+
+extension NotificationsTableViewCell {
+    struct Content: View {
+        private let title: String
+        private let description: String?
+        private let shouldShowIndicator: Bool
+        private let avatarStyle: AvatarsView.Style
+        private let actionIconName: String? // TODO: Will be refactored to contain the action.
+
+        init(
+            title: String,
+            description: String?,
+            shouldShowIndicator: Bool,
+            avatarStyle: AvatarsView.Style,
+            actionIconName: String?
+        ) {
+            self.title = title
+            self.description = description
+            self.shouldShowIndicator = shouldShowIndicator
+            self.avatarStyle = avatarStyle
+            self.actionIconName = actionIconName
+        }
+
+        var body: some View {
+            HStack(alignment: .top, spacing: 0) {
+                avatarHStack
+                textsVStack
+                    .offset(x: -avatarStyle.leadingOffset*2)
+                    .padding(.horizontal, Length.Padding.split)
+                if let actionIconName {
+                    actionIcon(withName: actionIconName)
+                }
+            }
+            .padding(.trailing, Length.Padding.double)
+        }
+
+        private var avatarHStack: some View {
+            HStack(spacing: 0) {
+                if shouldShowIndicator {
+                    indicator
+                        .padding(.horizontal, Length.Padding.single)
+                    AvatarsView(style: avatarStyle)
+                        .offset(x: -avatarStyle.leadingOffset)
+                } else {
+                    AvatarsView(style: avatarStyle)
+                        .offset(x: -avatarStyle.leadingOffset)
+                        .padding(.leading, Length.Padding.medium)
+                }
+            }
+        }
+
+        private var indicator: some View {
+            Circle()
+                .fill(Color.DS.Background.brand(isJetpack: AppConfiguration.isJetpack))
+                .frame(width: Length.Padding.single)
+        }
+
+        private var textsVStack: some View {
+            VStack(alignment: .leading, spacing: 2) {
+                Text(title)
+                    .style(.bodySmall(.regular))
+                    .foregroundStyle(Color.DS.Foreground.primary)
+                    .lineLimit(2)
+
+                if let description {
+                    Text(description)
+                        .style(.bodySmall(.regular))
+                        .foregroundStyle(Color.DS.Foreground.secondary)
+                        .lineLimit(2)
```

Comment:
The line limit is a bit complicated. I haven't found an elegant solution to calculate this in SwiftUI. One option was to calculate it via UILabel, by setting a frame and forcing it to layout. I don't want to spend too much time on this before we finish more tasks. If you have a proposal please lmk.